stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/Messages.kt

```diff
@@ -133,11 +135,13 @@ public fun Messages(
  *
  * @param messagesState The state of messages, current message list, thread, user and more.
  * @param lazyListState The scrolling state of the list, used to manipulate and trigger scroll events.
+ * @param focusedMessageOffset Scroll to focused item offset.
  */
 @Composable
 internal fun BoxScope.DefaultMessagesHelperContent(
     messagesState: MessagesState,
     lazyListState: LazyListState,
+    focusedMessageOffset: Int = DefaultFocusedMessageOffset
```

Comment:
This offset and all of its instances in parameters can easily be put in `MessagesState` so that we don't introduce another parameter to 10 components